lib/helpers/table-layer-metadata.js

```diff
@@ -23,14 +26,17 @@ class TableLayerMetadata {
 
     const {
       params: { layer: layerId } = {},
-      query = {},
-      ...otherOptions
-    } = options
+      query = {}
+    } = req
+
+    const currentVersion = _.get(req, 'app.locals.config.featureServer.currentVersion', CURRENT_VERSION)
+    const fullVersion = _.get(req, 'app.locals.config.featureServer.currentVersion', FULL_VERSION)
```

Comment:
should this be `fullVersion`?

Reply:
done in [06096f1](https://github.com/koopjs/FeatureServer/pull/230/commits/06096f136c66a9367e3127bc8205d7c8d2af6b35).